### PR TITLE
WIP Voice Demo page

### PR DESCRIPTION
**Pull request by JeffHowes**

### CODE REVIEW

File: src/hooks/useSpeechRecognition.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const languageOptions = [
     {label: 'Español', value: 'es-MX'},
     {label: 'Svenska - Swedish', value: 'sv-SE'}
 ];
+//This is just a test comment. JH
```
Comment (blazinaj) [inline src/hooks/useSpeechRecognition.js:21]: 👍

File: src/components/VoiceDemo/VoiceDemo.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect, useContext } from "react";
+import {VoiceCommandsContext} from "../../context/VoiceCommandsContext";
+
+const VoiceDemo = (props) => {
```
Comment (toha87) [inline src/components/VoiceDemo/VoiceDemo.js:5]: Good job Jeff!

File: src/components/VoiceDemo/VoiceDemo.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    const voiceContext = useContext(VoiceCommandsContext).SpeechRecognitionHook;
+
+    const whoIsTheManCommand = {
+        command: ["mirror mirror on the wall who is the man"],
```
Comment (toha87) [inline src/components/VoiceDemo/VoiceDemo.js:10]: I like it :)

**Review by toha87 — APPROVED** [review]
💯

**Review by blazinaj — APPROVED** [review]
Well done sir! 💯

File: src/hooks/useSpeechRecognition.js (hunk 1 above)
Comment (blazinaj) [inline src/hooks/useSpeechRecognition.js:21]: Might want to remove this comment now 😃

File: src/components/VoiceDemo/VoiceDemo.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        answer: "Stu is the man",
+        //func: () => ???
```
Comment (blazinaj) [inline src/components/VoiceDemo/VoiceDemo.js]: Should we display a modal popup with Stu's face?? 🤣

**Review by blazinaj — APPROVED** [review]
Well done, clean code! Good job using our context patterns

**Comment by JeffHowes** [comment]
Merging in Voice Demo page code

**Pull request merged**